configure.ac

```diff
@@ -1655,6 +1663,63 @@ if test "x$HAVE_ZLIB" = "xyes" -a "x$HAVE_ZLIB_H" = "xyes" -a "x$HAVE_COMPRESS2"
     EXTERNAL_FILTERS="${EXTERNAL_FILTERS}deflate(zlib)"
 fi
 
+## TODO - Probably this should be reworked or moved somewhere else
+
+# First arg has name and value. If first in undef, then non-empty second value is default
+m4_define([SETUP_API_TEST_VAR],
```

Comment:
What are these macros for? Reaching for m4 macros is probably very overkill

Reply:
`SETUP_API_TEST_VAR` is a helper to set up macros if an argument is provided at configuration time. If not, then they're either set to a default value or not defined. It's used for the new variables introduced to handle building the API tests with autotools.